src/users/interfaces/create-user-options.ts

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+export interface CreateUserOptions {
+  country_code: string;
```

Comment:
We use camelCase for internal options / types
```suggestion
  countryCode: string;
```